.github/workflows/coverage.yml

```diff
@@ -0,0 +1,22 @@
+name: Coverage
+
+on: [pull_request, push]
+
+jobs:
+  coverage:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_TERM_COLOR: always
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Rust
+        run: rustup toolchain install nightly-2022-05-09 --component llvm-tools-preview
```

Comment:
https://github.com/filecoin-project/ref-fvm/blob/0b2a24fdff8de8dac6aa0d262be719942ad9a2d8/.github/workflows/ci.yml#L18-L23

Reply:
(but with the right component.